src/lib/isIPRange.js

```diff
@@ -0,0 +1,22 @@
+import assertString from './util/assertString';
+
+const ipv4RangeMaybe = /^(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})\/(\d{1,2})$/;
+
+export default function isIPRange(str, version = '') {
```

Comment:
You might as well just drop the `version` param and make a note in the README that it checks IPv4 ranges only.